index.json

```diff
@@ -0,0 +1,37 @@
+[
```

Comment:
The index.json file created after testing the hugo-lunr plugin. This will be updated when the data for each source is updated.